travis.yml

```diff
@@ -0,0 +1,8 @@
+sudo: false
+
+language: node_js
+
+node_js:
+  - "node"
+
+after_success: npm run coverage
```

Comment:
You forgot the new line

Reply:
Done.. I'll make sure i remember next time